README.md

```diff
@@ -2,6 +2,62 @@
 
 A Kubernetes Operator based on the Operator SDK for creating and managing Grafana instances.
 
+
+## Companies that trust and use the Grafana operator:
+
+
+
+<table class="tg">
```

Comment:
Would a markdown table be easier to maintain here?

Something like

```markdown
|Team|Logo|
|--|--|
|Red Hat|logo|
|Integreatly|logo|
|Continental|logo|
```
|Team|Logo|
|--|--|
|Red Hat|logo|
|Integreatly|logo|
|Continental|logo|

etc..

Reply:
I'll look into it again, doing it initially I've had a bit of a formatting nightmare in terms of sizing.

Reply:
Coming back to this, I'd be more for using html, Found working with markdown tables an absolute naightmare, this would also mean that we stretch out the readme vertically (might be a bit of a nuisance with scrolling) using HTML we can format the table into a grid rather than list.